capella_console_client/config.py

```diff
@@ -94,3 +94,68 @@
     "local_time": "locale:time",
     "epsg": "proj:epsg",
 }
+
+
+TASKING_REQUEST_COLLECT_CONSTRAINTS_KEYS = frozenset(
+    [
+        "collect_mode",
+        "look_direction",
+        "asc_dsc",
+        "orbital_planes",
+        "local_time",
+        "off_nadir_min",
+        "off_nadir_max",
+        "elevation_min",
+        "elevation_max",
+        "image_length",
+        "image_width",
+        "azimuth",
+        "grr_min",
+        "grr_max",
+        "srr_min",
+        "srr_max",
+        "azr_min",
+        "azr_max",
+        "nesz_max",
+        "num_looks",
+        "polarization",
+    ]
+)
+
+
+REPEAT_REQUEST_COLLECT_CONSTRAINTS_KEYS = frozenset(
```

Comment:
`TASKING_REQUEST_COLLECT_CONSTRAINTS_KEYS` and `REPEAT_REQUEST_COLLECT_CONSTRAINTS_KEYS` look identical to me. 

if that's the case let's change this to

```python
COMMON_COLLECT_CONSTRAINTS_KEYS = frozenset(...)

TASKING_REQUEST_COLLECT_CONSTRAINTS_KEYS = frozenset([*COMMON_COLLECT_CONSTRAINTS_KEYS])

REPEAT_REQUEST_COLLECT_CONSTRAINTS_KEYS = frozenset([*COMMON_COLLECT_CONSTRAINTS_KEYS])
```

to allow a potential future where they'd diverge?